taichi/backends/opengl/aot_data.h

```diff
@@ -43,14 +43,35 @@ struct CompiledFieldData {
             column_num);
 };
 
+struct CompiledNdarrayData {
+  std::string field_name;
+  uint32_t dtype;
+  std::string dtype_name;
+  size_t dim{0};
+  std::vector<int> shape;
```

Comment:
Actually we don't want to restrict the shape here?

Reply:
Yup it's more like a FYI to show an example shape, `dim` info above is more important.

Reply:
Ack, let's just remove it :-)

Reply:
Can we remove `shape`?